Sources/Modifiers.swift

```diff
@@ -649,3 +649,80 @@ private final class PointerOfImpl<T : Arbitrary> : Arbitrary {
 	}
 }
 
+/**
+ Class used to generate values from mulitple `Gen` instances.
+ 
+ Given a StdGen and size, generate values from other generators, splitting the StdGen
+ after each call to `generate`, ensuring suffiicent entropy across generators.
```

Comment:
Could you please do a quick once over for spelling? "sufficiient" snuck through here but not elsewhere.